.github/workflows/main.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       run: make
     - name: check
       run: make check
+    - name: cache
+      run: make cache
```

Comment:
The target should not be `cache`. Instead, it would be `make tests`, which triggers the automated testing for all available tests. At the moment, cache specific items are associated.

Reply:
Does this means changing the `name` column to `tests` while maintaining the `make cache` command in the `run` field? After this change, other tests would put command in the `run` field.

Reply:
The name of GitHub Actions would accordingly be changed to `tests`.